Formula/llama.rb

```diff
@@ -0,0 +1,17 @@
+class Llama < Formula
+  desc "Terminal file manager"
+  homepage "https://github.com/antonmedv/llama"
+  url "https://github.com/antonmedv/llama/archive/refs/tags/v1.0.2.tar.gz"
+  sha256 "f0e306022b0073256b7c1f5856610a7bca5c9d3a7a2be07dde780d4038985968"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    assert_match "llama", pipe_output(bin/"llama", "--help").strip
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests.

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Unfortunately no, as this is a TUI lib to test we need a filesystem and PTY to simulate user behavior.

Reply:
Will it fail reliably without a filesystem? Because a PTY can be simulated.

Reply:
Yes, as llama is tui file manager.

Reply:
I can write a test same as in nnn file manager: https://github.com/Homebrew/homebrew-core/blob/8f38abd94169fd2dbf7efe2cabb6eae079c6352e/Formula/nnn.rb